secrets/browser-plugin/chrome/README.md

```diff
@@ -0,0 +1,33 @@
+# vaultPass
```

Comment:
Look like this README is from the original assets.  You might want to add a brief mention that you are leveraging the browser plugin developed by the XXXXX. 

```suggestion
These assets are provided to perform the tasks described in the [Vault Secrets in a Browser Plugin](https://learn.hashicorp.com/tutorials/vault/browser-plugin) tutorial.

-------

# vaultPass
```